### ISSUE TITLE
Unity 2019.3 FancyFolder fails when you zooming out in project view

### ISSUE THREAD

**Issue by reinholdsberzins** [Placeholder-Software/FancyFolders#3]
If you zoom out the Project Window to list not icon view whole folder system crashes and restart is needed to get the colored folder back.
![image](https://user-images.githubusercontent.com/24436198/71777579-95a84480-2faa-11ea-8065-6cf37e4c0060.png)

Here are the errors:
```
Unhandled Project Browser Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
  at FancyFolders.Editor.Renderer.ProjectBrowserRenderer.DrawDetailedItemInfo (System.String guid, UnityEngine.Rect rect) [0x000e6] in C:\Users\Reinh\OneDrive\FLR\FLR\Assets\Plugins\FancyFolders\Editor\Renderer\ProjectBrowserRenderer.cs:70 
UnityEngine.Debug:LogError(Object)
FancyFolders.Editor.Renderer.ProjectBrowserRenderer:DrawDetailedItemInfo(String, Rect) (at Assets/Plugins/FancyFolders/Editor/Renderer/ProjectBrowserRenderer.cs:75)
FancyFolders.Editor.<>c__DisplayClass6_0:<Initialize>b__2(String, Rect) (at Assets/Plugins/FancyFolders/Editor/Main.cs:63)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```
```
Disabling Fancy Folders Asset due to unhandled error. Please report this error at: https://github.com/Placeholder-Software/FancyFolders/issues
UnityEngine.Debug:LogError(Object)
FancyFolders.Editor.Renderer.ProjectBrowserRenderer:DrawDetailedItemInfo(String, Rect) (at Assets/Plugins/FancyFolders/Editor/Renderer/ProjectBrowserRenderer.cs:76)
FancyFolders.Editor.<>c__DisplayClass6_0:<Initialize>b__2(String, Rect) (at Assets/Plugins/FancyFolders/Editor/Main.cs:63)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

**Comment by martindevans** [comment]
I couldn't reproduce this. Before making any changes I think the other error (#4) happened first, after I had fixed that error this never happened - I suspect this error was the same root cause.

**Comment by reinholdsberzins** [comment]
This was fixed by #4 issue!
